build/build_release.sh

```diff
@@ -337,13 +337,16 @@ fi
 
 ln -sf ../../../doc/en/user/build/html user
 ln -sf ../../../doc/en/developer/build/html developer
+ln -sf ../../../doc/en/release/README.txt readme
```

Comment:
link this as README.txt or it will show up in the result as the file "readme". The "txt" extension helps those of us on primitive computers know we can open the file.